jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsSubsystemDefinition.java

```diff
@@ -52,9 +52,9 @@ public class JaxrsSubsystemDefinition extends SimpleResourceDefinition {
     public static final ModuleIdentifier JACKSON_DATATYPE_JDK8 = ModuleIdentifier.create("com.fasterxml.jackson.datatype.jackson-datatype-jdk8");
     public static final ModuleIdentifier JACKSON_DATATYPE_JSR310 = ModuleIdentifier.create("com.fasterxml.jackson.datatype.jackson-datatype-jsr310");
 
-    public static final ModuleIdentifier JAXB_API = ModuleIdentifier.create("javax.xml.bind.api");
-    public static final ModuleIdentifier JSON_API = ModuleIdentifier.create("javax.json.api");
-    public static final ModuleIdentifier JAXRS_API = ModuleIdentifier.create("javax.ws.rs.api");
+    public static final ModuleIdentifier JAXB_API = ModuleIdentifier.create("jakarta.xml.bind.api");
+    public static final ModuleIdentifier JSON_API = ModuleIdentifier.create("jakarta.json.api");
+    public static final ModuleIdentifier JAXRS_API = ModuleIdentifier.create("jakarta.ws.rs.api");
```

Comment:
Since these fields are public, I left them as they are without converting them to Strings

Reply:
Is the module public or private?

Reply:
Good hint, the module is private, so, we are able to modify these fields without worrying about public usage